src/journal-remote/microhttpd-util.c

```diff
@@ -60,7 +60,7 @@ static int mhd_respond_internal(struct MHD_Connection *connection,
         if (!response)
                 return MHD_NO;
 
-        log_debug("Queing response %u: %s", code, buffer);
+        log_debug("Queueing response %u: %s", code, buffer);
```

Comment:
2016-07-12 11:50 GMT+02:00 Lennart Poettering <notifications@github.com>:

> In src/journal-remote/microhttpd-util.c
> <https://github.com/systemd/systemd/pull/3705#discussion_r70408310>:
>
> > @@ -60,7 +60,7 @@ static int mhd_respond_internal(struct MHD_Connection *connection,
> >          if (!response)
> >                  return MHD_NO;
> >
> > -        log_debug("Queing response %u: %s", code, buffer);
> > +        log_debug("Queueing response %u: %s", code, buffer);
>
> this should be "Queuing", no? i.e. both the original and the fix are
> wrong, no?
>
>
 Both are correct according to
http://www.merriam-webster.com/dictionary/Queueing

I can change it though, if you want.


-- 
Why is it that all of the instruments seeking intelligent life in the
universe are pointed away from Earth?